Makefile

```diff
@@ -157,9 +157,12 @@ verify: .init .generate_files
 	@rm .out
 	@#
 	@echo Running golint and go vet:
-	# Exclude the generated (zz) files for now
+	@# Exclude the generated (zz) files for now, as well as defaults.go (it
```

Comment:
I expanded the scope of the comment, and then decided to hide it because it looked like TMI. Do you want me to unhide just the bit that was there before? Or all of it?